docs/contributing-guide/tools.md

```diff
@@ -0,0 +1,43 @@
+# Prerequisites and tools
+
+Here is a list of the tools used to develop the project, followed by details on each tool:
+
+- **Required:** [`node.js`](https://nodejs.org/en/download/prebuilt-installer) (Preferably managed by [`volta`](https://docs.volta.sh/guide/getting-started))
+- **Required:** [Commit signing configured](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits), without it, the pull requests **WILL NOT BE ACCEPTED!**
+- _Recommended:_ [VS Code](https://code.visualstudio.com/Download)
+- _Optional:_ [Resend account](https://resend.com) (Used for sending emails)
+
+## `node.js` and `volta`
+
+Most of the tools used are based on `node.js`, so you should have it installed on your machine. It may be installed from an installer, or a version manager for node.
+
+Our recommendation is using `volta` for managing the node versions, it is already configured for the project, so you only need to install `volta` on your machine and it will take care of downloading the correct version of node when you first try installing the dependencies.
+
+If you already have `nvm` installed we recommend you uninstall since it may conflict with `volta`.
```

Comment:
Question.. the project now has a .nvmrc so is this still relevant?

Reply:
I think sorta...

The `.nvmrc` file helps with cloudflare as it reads that to determine the node version. And it is set to use the latest LTS version, so both when building on cloudflare and if you use nvm it will always point to the latest LTS.

Volta keeps track using the `volta` field on `package.json`. It is set to a version, and not a moving target, like "LTS".

So, we can say we support both tools, but they should not be both installed in the user's system, and we should also update the versions so both match (and move the nvm version to a number instead of the moving target).